src/libsyncengine/update_detection/file_system_observer/localfilesystemobserverworker.cpp

```diff
@@ -180,16 +203,16 @@ void LocalFileSystemObserverWorker::changesDetected(const std::list<std::pair<st
                 if (!itemId.empty()) {
                     // Remove it from snapshot
                     _snapshot->removeItem(itemId);
-                    LOGW_SYNCPAL_DEBUG(_logger, L"Item removed from sync because it's hidden: "
-                                                        << Utility::formatSyncPath(absolutePath).c_str());
+                    LOGW_SYNCPAL_DEBUG(_logger,
+                                       L"Item removed from sync because it's hidden: " << Utility::formatSyncPath(absolutePath));
                 } else {
-                    LOGW_SYNCPAL_DEBUG(_logger, L"Item not processed because it's excluded: "
-                                                        << Utility::formatSyncPath(absolutePath).c_str());
+                    LOGW_SYNCPAL_DEBUG(_logger,
+                                       L"Item not processed because it's excluded: " << Utility::formatSyncPath(absolutePath));
```

Comment:
```suggestion
                                       L"Item not processed because it is excluded: " << Utility::formatSyncPath(absolutePath));
```

Reply:
Why not....

Reply:
It is subjective: simpler to read, do not collide with the single quotes used as path delimiters.

Reply:
Done